lib/workers/repository/config-migration/branch/rebase.ts

```diff
@@ -15,8 +15,9 @@ export async function rebaseMigrationBranch(
   migratedConfigData: MigratedData,
 ): Promise<string | null> {
   logger.debug('Checking if migration branch needs rebasing');
+  const baseBranch = config.defaultBranch!;
```

Comment:
@rarkins should we use `baseBranch` instead of `defaultBranch` here? do we support config migration on multiple base branches? 🤔

Reply:
Do we migrate configs in base branches? It's possible we only migrate in the default branch so this is correct

Reply:
Based on my reading of https://github.com/renovatebot/renovate/blob/4129ab2077a28c551bf8d48d8d80388a8a741289/lib/workers/repository/config-migration/pr/index.ts#L101, I understand that it's only the default branch.

Reply:
Yes I think it's only default branch, so a hardcoded default branch here is ok